text/src/autogluon/text/automm/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -74,3 +90,12 @@ model:
     drop_rate: 0.1
     normalization: "layer_norm"
     data_types:
+  # fusion_transformer:
+  #   hidden_size: 192
+  #   n_blocks: 3
+  #   attention_n_heads: 8
+  #   adapt_in_features: "max"
+  #   attention_dropout: 0.2
+  #   residual_dropout: 0.0
+  #   ffn_dropout: 0.1 
+  #   ffn_d_hidden: 192  
```

Comment:
Why comment these sections?

Reply:
The `models.names` have to exactly match with the `models.{MODEL NAME}`, otherwise an error will be raised.

Reply:
What kind of error message are you seeing?

Reply:
It reads like :

`assert sorted(config.names) == sorted(keys),\` 
`AssertionError: ['hf_text', 'timm_image', 'clip', 'categorical_transformer', 'numerical_transformer', 'fusion_transformer'] do not match config keys ['categorical_mlp', 'categorical_transformer', 'numerical_transformer', 'hf_text', 'timm_image', 'clip', 'fusion_transformer']`

Reply:
Maybe we can try using `set(config.names).issubset(set(keys))` as a replacement for `sorted(config.names) == sorted(keys)`, as it can guarantee each model in config.names has an associated model_config.

Reply:
Yeah, this is also feasible.